packages/editor/src/lib/editor/Editor.ts

```diff
@@ -6988,7 +6992,10 @@ export class Editor extends EventEmitter<TLEventMap> {
 	 *
 	 * @public
 	 */
-	createShapes<T extends TLUnknownShape>(shapes: OptionalKeys<TLShapePartial<T>, 'id'>[]): this {
+	createShapes<T extends TLUnknownShape>(
+		shapes: OptionalKeys<TLShapePartial<T>, 'id'>[],
+		select = false
```

Comment:
Docs above mention this param, but the function didn't actually define it.

Reply:
i think we removed this a while ago because it complicates the API and it's pretty straightforward to do create then select. mind if we leave it out still?

Reply:
Happy to remove 👍